### PR TITLE
Fix devtools popup inspection

### PR DESCRIPTION
**Pull request by pr8x**
Seems that this got broken recently. It checks if `Root` (`Application`) is a `TopLevel` which could never work. Instead we have to get the hovered `TopLevel` and try to find its "root" `TopLevel` (which is the root `Window` on desktop).

### CODE REVIEW

File: src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs [hunk 1]
```diff
@@ -161,8 +161,19 @@ private void RawKeyDown(RawKeyEventArgs e)
...
 
-            var root = Root as TopLevel
-                ?? vm.PointerOverRoot as TopLevel;
+            var root = vm.PointerOverRoot as TopLevel;
+            IControl? parent = root;
+
+            while (parent?.Parent != null)
+            {
+                parent = parent.Parent;
+            }
```
Comment (maxkatz6) [inline src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs:170]: Why not to use VisualRoot property on control?

Comment (pr8x) [reply src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs:170]: Not sure why, but `VisualRoot `on the `PopupRoot` was always null. I guess I could somehow access the `_parent` field of `PopupRoot`?

Comment (pr8x) [reply src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs]: I exposed the field now. That seems to be the cleanest solution here IMO.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019582-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by jmacato — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0019607-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**